src/Workspaces/CoreTestUtilities/WorkspacesTestUtilities.csproj

```diff
@@ -18,6 +18,7 @@
     <ProjectReference Include="..\..\Compilers\CSharp\Portable\CSharpCodeAnalysis.csproj" />
     <ProjectReference Include="..\..\Compilers\Test\Resources\Core\CompilerTestResources.csproj" />
     <ProjectReference Include="..\..\Compilers\VisualBasic\Portable\BasicCodeAnalysis.vbproj" />
+    <ProjectReference Include="..\..\EditorFeatures\Core\EditorFeatures.csproj" />
```

Comment:
This feels like a layering violation here: can we have the IForegroundNotificationService somehow add itself to a list of "things that need cleanup" for the UseExportProviderAttribute?

Reply:
➡️ Filed #25805. I have ideas for how to handle this but don't want to block on it due to scale of the work.

Reply:
I guess I want to make sure that isn't viewed as optional, as this is opening a barn door and the animals are ready to escape.